paz/models/detection/frustum_pointnet.py

```diff
@@ -0,0 +1,177 @@
+import tensorflow as tf
+from tensorflow.keras.layers import Conv2D, Dense
+from tensorflow.keras.layers import Dropout
+from tensorflow.keras.layers import Input, Lambda
+from tensorflow.keras.layers import MaxPooling2D
+from tensorflow.keras.models import Model
+
+from paz.models.detection.model_util import NUM_HEADING_BIN, NUM_SIZE_CLUSTER
+from paz.models.detection.model_util import parse_output_to_tensors
+from paz.models.detection.model_util import point_cloud_masking
+from paz.optimization.losses.frustumpointnet_loss import FPointNet_loss
+
+
+def Instance_Seg_Net(point_cloud, one_hot_vec):
```

Comment:
Add documentation using the PAZ convention (just look at how we do it for other models)